app/app_iface.go

```diff
@@ -170,6 +170,8 @@ type AppIface interface {
 	GetClusterPluginStatuses() (model.PluginStatuses, *model.AppError)
 	// GetConfigFile proxies access to the given configuration file to the underlying config store.
 	GetConfigFile(name string) ([]byte, error)
+	// GetEditHistoryForPost response bool value indicates, if the post is inaccessible due to cloud plan's limit.
```

Comment:
Can you change the comment to match the correct response types?

Reply:
Updated